helm/presets/aws.yaml

```diff
@@ -0,0 +1,44 @@
+receive:
+  service:
+    type: LoadBalancer
+broadcast:
+  service:
+    type: LoadBalancer
+acquire:
+  service:
+    type: LoadBalancer
+strimzi:
+  kafka:
+    resources:
+      requests:
+        cpu: 500m
+        memory: 6Gi
+      limits:
+        cpu: 1000m
+        memory: 8Gi
+  zookeeper:
+    resources:
+      requests:
+        cpu: 100m
+        memory: 512Mi
+      limits:
+        cpu: 250m
+        memory: 1Gi
+redis:
+  enabled: false
```

Comment:
want to enable this for now?  Just because you put it in aws doesn't mean you want a managed service redis.

Reply:
No I want the presets to assume you're using managed services for Redis and Postgres. You can always override this at the command line.